tracer/build/_build/Build.Gitlab.cs

```diff
@@ -102,9 +102,52 @@ void SignBinary(AbsolutePath binaryPath)
                     logOutput: false,
                     logInvocation: false);
             signProcess.WaitForExit();
+
+            var output = signProcess.Output.Select(o => o.Text);
+            foreach (var line in output)
+            {
+                Logger.Information("[dd-wcs] {Line}", line);
+
+                // dd-wcs will return 0 even if there are errors
+                if (line.StartsWith("ERROR:", StringComparison.OrdinalIgnoreCase))
+                {
+                    throw new Exception($"Error found when signing {binaryPath}: {line}");
+                }
```

Comment:
And if I understood correctly you were able to test this out locally right?

Reply:
Yes, I was getting some error messages in the tool's log starting with the word "ERROR:". The tool would return 0 though.